qa/suites/rados/singleton/all/osd-recovery.yaml

```diff
@@ -22,6 +22,7 @@ tasks:
       - \(OBJECT_DEGRADED\)
       - \(SLOW_OPS\)
       - \(REQUEST_SLOW\)
+      - slow requests
```

Comment:
could you add a
```
(cherry picked from commit 0e987665fe480530a0d1664392604713828bfa5c)
```

 line with a `Conflicts:` section? so we can connect it to the change in master, and explain that this change is adapted to luminous ?